srcs/parsing/validation.c

```diff
@@ -1,68 +1,10 @@
 #include "minishell.h"
 
-// deal with -eeeeeeeeeee or -aaaaaaaaaaaaaaaa 
-
-
-static int	get_args_count(t_token *tokens, int nb_tokens)
-{
-	int count;
-	int	i;
-
-	count = 0;
-	i = 0;
-	while (i < nb_tokens)
-	{
-		if (tokens[i].type == WORD)
-			count++;
-		i++;
-	}
-	return (count);
-}
-
-static int	get_flags_count(t_token *tokens, int nb_tokens)
-{
-	int count;
-	int	i;
-
-	count = 0;
-	i = 0;
-	while (i < nb_tokens)
-	{
-		if (tokens[i].type == FLAG)
-			count++;
-		i++;
-	}
-	return (count);
-}
-
-void	assign_tokens(t_cmd_table *cmd_table)
-{
-	int	i;
-	int	j;
-	int f;
-	int	nb_args;
-	int	nb_flags;
-
-	i = 0;
-	j = 0;
-	f = 0;
-	nb_args = get_args_count(cmd_table->tokens, cmd_table->nb_tokens);
-	cmd_table->cmd_args = calloc_or_exit(sizeof(char *), nb_args + 1);
-	nb_flags = get_flags_count(cmd_table->tokens, cmd_table->nb_tokens);
-	cmd_table->flags = calloc_or_exit(sizeof(char*), nb_flags + 1);
-	while (i < cmd_table->nb_tokens)
-	{
-		if (cmd_table->tokens[i].type == CMD)
-			cmd_table->cmd_name = cmd_table->tokens[i].text;
-		else if (cmd_table->tokens[i].type == FLAG)
-			cmd_table->flags[f++] = cmd_table->tokens[i].text;
-		else if (cmd_table->tokens[i].type == WORD)
-			cmd_table->cmd_args[j++] = cmd_table->tokens[i].text;
-		i++;
-	}
-	if (nb_args)
-		cmd_table->cmd_args[j] = 0;
-	if (nb_flags)
-		cmd_table->flags[f] = 0;
-	return ;
-}
\ No newline at end of file
+// deal with -eeeeeeeeeee or -aaaaaaaaaaaaaaaa or see if the executor handles it.

```

Comment:
Was this done somewhere else?

Reply:
no. I think the executor will handle it actually.
Don't pay attention to this file. It's there but I think I'll delete it it's quite useless at the moment.